[BUG] compile failed in databricks runtimes due to new added TestReport

**Describe the bug**
related to new merged https://github.com/NVIDIA/spark-rapids/pull/9089

```
[2023-08-31T10:28:17.143Z] [ERROR] /home/ubuntu/spark-rapids/integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala:22: object json4s is not a member of package org

[2023-08-31T10:28:17.143Z] [ERROR] import org.json4s._

[2023-08-31T10:28:17.143Z] [ERROR]            ^

[2023-08-31T10:28:17.143Z] [ERROR] /home/ubuntu/spark-rapids/integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala:23: object json4s is not a member of package org

[2023-08-31T10:28:17.143Z] [ERROR] import org.json4s.jackson.Serialization.writePretty

[2023-08-31T10:28:17.143Z] [ERROR]            ^

[2023-08-31T10:28:17.143Z] [ERROR] /home/ubuntu/spark-rapids/integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala:41: not found: value DefaultFormats

[2023-08-31T10:28:17.143Z] [ERROR]     implicit val formats = DefaultFormats

[2023-08-31T10:28:17.143Z] [ERROR]                            ^

[2023-08-31T10:28:17.143Z] [ERROR] /home/ubuntu/spark-rapids/integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala:43: not found: value writePretty

[2023-08-31T10:28:17.143Z] [ERROR]     os.write(writePretty(queryMetas).getBytes)

[2023-08-31T10:28:17.143Z] [ERROR]              ^

[2023-08-31T10:28:17.143Z] [ERROR] /home/ubuntu/spark-rapids/integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala:22: Unused import

[2023-08-31T10:28:17.143Z] [ERROR] import org.json4s._

[2023-08-31T10:28:17.143Z] [ERROR]                   ^

[2023-08-31T10:28:17.143Z] [ERROR] /home/ubuntu/spark-rapids/integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala:23: Unused import

[2023-08-31T10:28:17.143Z] [ERROR] import org.json4s.jackson.Serialization.writePretty

[2023-08-31T10:28:17.143Z] [ERROR]                                         ^

[2023-08-31T10:28:17.143Z] [ERROR] /home/ubuntu/spark-rapids/integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/TestReport.scala:41: local val formats in method save is never used

[2023-08-31T10:28:17.143Z] [ERROR]     implicit val formats = DefaultFormats

[2023-08-31T10:28:17.143Z] [ERROR]                  ^

[2023-08-31T10:28:17.143Z] [ERROR] 7 errors found
```

**Steps/Code to reproduce bug**
Please provide a list of steps or a code sample to reproduce the issue.
Avoid posting private or sensitive data.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Environment details (please complete the following information)**
 - Environment location: [Standalone, YARN, Kubernetes, Cloud(specify cloud provider)]
 - Spark configuration settings related to the issue

**Additional context**
Add any other context about the problem here.

--- a/integration_tests/pom.xml
+++ b/integration_tests/pom.xml
@@ -289,6 +289,24 @@
                     <scope>provided</scope>
                 </dependency>
                 <dependency>
+                    <groupId>org.json4s</groupId>
+                    <artifactId>json4s-ast_${scala.binary.version}</artifactId>
+                    <version>${spark.version}</version>
+                    <scope>provided</scope>
+                </dependency>
+                <dependency>
+                    <groupId>org.json4s</groupId>
+                    <artifactId>json4s-core_${scala.binary.version}</artifactId>
+                    <version>${spark.version}</version>
+                    <scope>provided</scope>
+                </dependency>
+                <dependency>
+                    <groupId>org.json4s</groupId>
+                    <artifactId>json4s-jackson_${scala.binary.version}</artifactId>
+                    <version>${spark.version}</version>
+                    <scope>provided</scope>
+                </dependency>
+                <dependency>
                     <groupId>org.apache.commons</groupId>
                     <artifactId>commons-io</artifactId>
                     <version>${spark.version}</version>

--- a/jenkins/databricks/install_deps.py
+++ b/jenkins/databricks/install_deps.py
@@ -118,6 +118,8 @@ def define_deps(spark_version, scala_version):
                  f'{prefix_ws_sp_mvn_hadoop}--org.json4s--json4s-ast_{scala_version}--org.json4s__json4s-ast_{scala_version}__*.jar'),
         Artifact('org.json4s', f'json4s-core_{scala_version}',
                  f'{prefix_ws_sp_mvn_hadoop}--org.json4s--json4s-core_{scala_version}--org.json4s__json4s-core_{scala_version}__*.jar'),
+        Artifact('org.json4s', f'json4s-jackson_{scala_version}',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.json4s--json4s-jackson_{scala_version}--org.json4s__json4s-jackson_{scala_version}__*.jar'),
         Artifact('org.javaassist', 'javaassist',
                  f'{prefix_ws_sp_mvn_hadoop}--org.javassist--javassist--org.javassist__javassist__*.jar'),
         Artifact('com.fasterxml.jackson.core', 'jackson-core',

